gcovr/writer/json.py

```diff
@@ -297,6 +312,10 @@ def _line_from_json(json_line: dict) -> LineCoverage:
 
     insert_decision_coverage(line, _decision_from_json(json_line.get("gcovr/decision")))
 
+    if "gcovr/calls" in json_line:
+        for callno, json_call in enumerate(json_line["gcovr/calls"]):
+            insert_call_coverage(line, callno, _call_from_json(json_call))
```

Comment:
This callno doesn't match the original one. In the test data we have e.g.:
```html
            <div class="takenBranch">&check; Call 0 invoked.</div>
            <div class="notTakenBranch">&cross; Call 3 not invoked.</div>
            <div class="takenBranch">&check; Call 6 invoked.</div>
            <div class="notTakenBranch">&cross; Call 9 not invoked.</div>
```

It seems, that the `callno` should be saved in `CallCoverage` class and write to/read from JSON.

Reply:
Indeed. Thank you very much!
Changed to preserve the callno and updated the json format version to 0.4